google-http-client/src/test/java/com/google/api/client/http/HttpRequestFactoryTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2021 Google Inc.
```

Comment:
No (c)
In fact, never c in parentheses anywhere. It is *not* a legal copyright symbol

Reply:
done